index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="top">
+        <h2 class="logo">FABIEN</h2>
+        <button class="ham" type="button">
+          <img src="./img/Union.png" alt="hamburger img" />
+        </button>
```

Comment:
- [ ] Your hamburger menu should be wrapped with a `nav` tag. instead of using a `button` tag, Kindly replace it with a `nav` tag.